File: ChuckNorrisIO/app/src/main/java/br/com/titoschmidt/chucknorrisio/models/Joke.java
package br.com.titoschmidt.chucknorrisio.models;

import com.google.gson.annotations.SerializedName;

public class Joke {

    @SerializedName("icon_url")
    private final String iconUrl;
    @SerializedName("value")
    private final String value;

    public Joke(String iconUrl, String joke) {
        this.iconUrl = iconUrl;
        this.value = joke;
    }

    public String getIconUrl() {
        return iconUrl;
    }

    public String getValue() {
        return value;
    }
}

File: JokerApp/app/src/main/java/co/titoschmidt/tutorial/jokerappdev/model/Joke.kt
package co.titoschmidt.tutorial.jokerappdev.model

import com.google.gson.annotations.SerializedName

/*
 * 13/12/2021
 * @author titoesa@gmail.com (Tito Schmidt).
 * 
 */
data class Joke(

    @SerializedName("value") val text : String,
    @SerializedName("icon_url") val iconUrl : String
)
